lib/dns/server.js

```diff
@@ -321,15 +323,35 @@ class RootServer extends DNSServer {
       // TLD '._synth' is being queried on its own, send SOA
       // so recursive asks again with complete synth record.
       if (labels.length === 1) {
+        // Empty non-terminal proof:
+        res.authority.push(nsec.create('_synth.',
+          '\\000._synth.', TYPE_MAP_EMPTY));
```

Comment:
```suggestion
        res.authority.push(
          nsec.create(
            '_synth.',
            '\\000._synth.',
            TYPE_MAP_EMPTY
          )
        );
```

Just a style suggestion